RELEASES.md

```diff
@@ -1,3 +1,29 @@
+<a name="0.21.0"></a>
+## 0.21.0 (2024-05-30)
+
+Since the last release, a fair number of the commits have been focussed on cleaning
+up and improving LALRPOP's documentation. Shout out to Yudai Takada, George
+White, and Dinu Blanovschi.
+
+#### Bugfixes
+
+* A long-standing bug where LALRPOP would throw a "no entry found for key"
+  exception when trying to handle a particular reduce/reduce conflict has been
+  resolved.
```

Comment:
```suggestion
  exception when trying to handle certain grammars has been resolved.
```

I don't love losing specificity, but I think reduce/reduce conflict is misleading.  This is a reduce/reduce conflict from a LALR perspective that may or may not be a reduce/reduce conflict after we split the states to full LR(1).  So from a user perspective, they could hit this error when their grammar actually has a reduce/reduce conflict, or with a valid grammar that is LR(1) but not LALR.  I'm not sure how to capture all that without going into too much detail, but I think as originally written it implies that every time we hit that panic, the end result should have been a reduce/reduce conflict error, which isn't the case.